pkg/controller/volume/attachdetach/attach_detach_controller_test.go

```diff
@@ -227,9 +233,9 @@ func attachDetachRecoveryTestCase(t *testing.T, extraPods1 []*v1.Pod, extraPods2
 		csiNodeInformer.GetIndexer().Add(&csiNodeToAdd)
 	}
 
-	informerFactory.Start(stopCh)
+	informerFactory.Start(ctx.Done())
```

Comment:
Are we canceling the context here by using `ctx.Done()`? How this will affect to continue the rest of the flow?

Reply:
`Done()` does not cancel a context, it returns a channel that can be used to _detect_ the context is cancelled in the future.